package.nls.de.json

```diff
@@ -1,4 +1,5 @@
 {
+  "command.activateIcons.title": "",
```

Comment:
According to `vscode` translation for `Icon Theme` I'm think something in the line of `Dateisymboldesign aktivieren`.